src/main/java/org/bitbuckets/robot/RobotSetup.java

```diff
@@ -45,7 +44,7 @@ public RobotContainer build(ProcessPath path) {
         }
 
 
-        return new RobotContainer(driveSubsystem, armSubsystem, elevatorSubsystem);
+        return new RobotContainer(driveSubsystem, armSubsystem, elevatorSubsystem, visionControl);
```

Comment:
VisionControl not be on the same level as a subsystem, but we don't have to fix that on this branch.

Reply:
we found it!

Reply:
?